workflow/rules/map.smk

```diff
@@ -12,8 +12,8 @@ rule minimap2_bam:
     params:
         extra=get_minimap2_mapping_params,  # optional
         sorting="coordinate",  # optional: Enable sorting. Possible values: 'none', 'queryname' or 'coordinate'
-        sort_extra=lambda wc: f"-@ {workflow.cores * 0.5}",  # optional: extra arguments for samtools/picard
-    threads: workflow.cores * 0.5
+        sort_extra=lambda wc: f"-@ 4",  # optional: extra arguments for samtools/picard
```

Comment:
Is this hard-coded `4` intentional? Or should this also be `workflow.cores //2` as below?

Reply:
OK, had a look at the minimap2 docs and reduced the indexing threads to 3, as minimap 2 doesn't use more for indexing. Also, made the `sort_extra` lambda expression further depend on `threads`.